store/mongo/migration_2_0_0.go

```diff
@@ -0,0 +1,275 @@
+	"github.com/mendersoftware/go-lib-micro/mongo/migrate"
+	"github.com/mendersoftware/go-lib-micro/mongo/oid"
+	mstorev1 "github.com/mendersoftware/go-lib-micro/store"
+	mstore "github.com/mendersoftware/go-lib-micro/store/v2"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+const (
+	findBatchSize = 255
+)
+
+type migration_2_0_0 struct {
+	ds  *DataStoreMongo
+	ctx context.Context
+}
+
+var DbDevicesCollectionIndices = []mongo.IndexModel{
+	{
+		Keys: bson.D{
+			{Key: mstore.FieldTenantID, Value: 1},
+			{Key: dbFieldIDDataSha, Value: 1},
+		},
+		//nolint:staticcheck // SA1019
+		Options: mopts.Index().
+			SetName(strings.Join([]string{
+				mstore.FieldTenantID,
+				dbFieldIDDataSha,
+			}, "_")).
+			SetUnique(true).
+			SetBackground(false),
+	},
+	{
+		Keys: bson.D{
+			{Key: mstore.FieldTenantID, Value: 1},
+			{Key: dbFieldStatus, Value: 1},
+			{Key: dbFieldID, Value: 1},
+		},
+		//nolint:staticcheck // SA1019
+		Options: mopts.Index().
+			SetName(strings.Join([]string{
+				mstore.FieldTenantID,
+				dbFieldStatus,
+				dbFieldID,
+			}, "_")).
+			SetBackground(false),
+	},
+}
+
+var DbAuthSetsCollectionIndices = []mongo.IndexModel{
+	{
+		Keys: bson.D{
+			{Key: mstore.FieldTenantID, Value: 1},
+			{Key: dbFieldIDDataSha, Value: 1},
+			{Key: dbFieldPubKey, Value: 1},
+		},
+		Options: mopts.Index().
+			SetName(strings.Join([]string{
+				mstore.FieldTenantID,
+				dbFieldIDDataSha,
+				dbFieldPubKey,
+			}, "_")).
+			SetUnique(true),
+	},
+	{
+		Keys: bson.D{
+			{Key: mstore.FieldTenantID, Value: 1},
+			{Key: dbFieldDeviceID, Value: 1},
+		},
+		Options: mopts.Index().
+			SetName(strings.Join([]string{
+				mstore.FieldTenantID,
+				dbFieldDeviceID,
+			}, "_")),
+	},
+}
+
+var DbLimitsCollectionIndices = []mongo.IndexModel{
+	{
+		Keys: bson.D{
+			{Key: mstore.FieldTenantID, Value: 1},
+			{Key: dbFieldName, Value: 1},
+		},
+		Options: mopts.Index().
+			SetName(mstore.FieldTenantID + "_" + dbFieldName).
+			SetUnique(true),
+	},
+}
+
+var DbTokensCollectionIndices = []mongo.IndexModel{
+	{
+		Keys: bson.D{
+			{Key: mstore.FieldTenantID, Value: 1},
+			{Key: dbFieldID, Value: 1},
+		},
+		Options: mopts.Index().
+			SetName(mstore.FieldTenantID + "_" + dbFieldID),
+	},
+	{
+		Keys: bson.D{
+			{Key: mstore.FieldTenantID, Value: 1},
+			{Key: dbFieldExpTime, Value: 1},
+		},
+		Options: mopts.Index().
+			SetName(mstore.FieldTenantID + "_" + dbFieldExpTime),
+	},
+}
+
+// Up creates an index on status and id in the devices collection
+func (m *migration_2_0_0) Up(from migrate.Version) error {
+	ctx := context.Background()
+	client := m.ds.client
+
+	collections := map[string]struct {
+		Indexes []mongo.IndexModel
+	}{
+		DbAuthSetColl: {
+			Indexes: DbAuthSetsCollectionIndices,
+		},
+		DbDevicesColl: {
+			Indexes: DbDevicesCollectionIndices,
+		},
+		DbLimitsColl: {
+			Indexes: DbLimitsCollectionIndices,
+		},
+		DbTokensColl: {
+			Indexes: DbTokensCollectionIndices,
+		},
```

Comment:
Can we please skip migrating documents for this collection?

Reply:
sure (I nswered this comment but I have no idea why it is not visible here)